### ISSUE TITLE
producer_plugin should be able to modify subjective CPU billing parameters

### ISSUE THREAD

**Issue by arhag** [EOSIO/eos#3122]
producer_plugin should be able to modify the default CPU leeway.

**Comment by taokayan** [comment]
https://github.com/EOSIO/eos/pull/4410

**Comment by brianjohnson5972** [comment]
@taokayan did #4410 complete this issue?

**Comment by taokayan** [comment]
I think so. as #4410 add the ability to change the subjective cpu leeway:
```
   if (options.subjective_cpu_leeway_us) {
      chain::controller& chain = app().get_plugin<chain_plugin>().chain();
      chain.set_subjective_cpu_leeway(fc::microseconds(*options.subjective_cpu_leeway_us));
   }
```

**Comment by brianjohnson5972** [comment]
Resolved by #4410
